webapp/src/component/security/ResetPasswordSetView.tsx

```diff
@@ -16,14 +16,13 @@ import LoadingButton from 'tg.component/common/form/LoadingButton';
 import { Alert } from '../common/Alert';
 import { StandardForm } from '../common/form/StandardForm';
 import { DashboardPage } from '../layout/DashboardPage';
-import { SetPasswordFields } from './SetPasswordFields';
+import { NewPasswordLabel, SetPasswordField } from './SetPasswordField';
 import { useLogout } from 'tg.hooks/useLogout';
 
 const globalActions = container.resolve(GlobalActions);
 
 type ValueType = {
   password: string;
-  passwordRepeat: string;
```

Comment:
Why was this removed? It generally is common practice to have a double check measure to ensure there are no accidental typos in the password (if the user types their passwords manually).

I do know there's a whole debate about that within the UX community, that a single field is less work and therefore better for the user; but another dimension of UX is being helpful to the user and avoid letting them do mistakes and help correct them.

Reply:
Users don't type passwords manually anymore. Repeating passwords is deprecated. You can always reset it if you make a mistake.